src/modules/feed/components/Transaction.js

```diff
@@ -20,6 +21,12 @@ export class Transaction extends Component {
     this.state = {
       imageLoading: true
     };
+    if (this.props.transaction["receiver"] != undefined && this.props.transaction["sender"] != undefined) {
+      let receiver = this.props.transaction["receiver"];
+      let sender = this.props.transaction["sender"];
+      this.state.avatarReceiver = receiver["avatar-url"];
```

Comment:
This is not the react way to update state for various reasons. Always use `setState({})` here.